src/core/actions/categories.js

```diff
@@ -1,13 +1,15 @@
 import {
   CATEGORIES_FETCH,
   CATEGORIES_LOAD,
-  CATEGORIES_FAIL,
 } from 'core/constants';
 
-export function categoriesFetch() {
+export function categoriesFetch({ errorHandlerId } = {}) {
+  if (!errorHandlerId) {
```

Comment:
When we use Flow 100% we won't need it. We could also try [invariant](https://github.com/zertosh/invariant) in the meantime but it will still require the boilerplate code.